tools/tuf/test/README.md

```diff
@@ -36,6 +36,7 @@ ENROLL_SECRET=6/EzU/+jPkxfTamWnRv1+IJsO4T9Etju \
 FLEET_DESKTOP=1 \
 USE_FLEET_SERVER_CERTIFICATE=1 \
 DEBUG=1 \
+GOARCH=arm64 \ # defaults to amd64
```

Comment:
thinking outloud,  maybe I'm being overly cautious: should we move this below along with the `*_TEST_EXTENSIONS` vars? in my experience people copy and paste this to get started, so we'll be implicitly forcing them to compile arm64.

Reply:
Ah yeah if that's the most common path then worth changing. Done!